packages/theme/bin/generate-primitive-tokens/index.ts

```diff
@@ -22,26 +22,19 @@ const __dirname = path.dirname( __filename );
 // Path to the color.json file
 const colorJsonPath = path.join( __dirname, '../../tokens/color.json' );
 
-/**
- * Rounds a given hex value (0-255) to 3 decimal places.
- *
- * 3 decimal places is the minimum precision for lossless hex serialization.
- * With 3 decimal places rounding to the nearest 0.001, the maximum rounding
- * error is 0.0005. With 256 possible hex values, 0.0005 × 256 = 0.128,
- * guaranteeing the rounded value stays within 0.5 of the original value.
- *
- * @param n - The hex value to round.
- * @return The rounded hex value.
- */
-const roundHexComponent = ( n: number ) => Math.round( n * 1000 ) / 1000;
+// 3 decimal places is the minimum precision for lossless hex serialization.
+// With 3 decimal places rounding to the nearest 0.001, the maximum rounding
+// error is 0.0005. With 256 possible hex values, 0.0005 × 256 = 0.128,
+// guaranteeing the rounded value stays within 0.5 of the original value.
+const ROUNDING_PRECISION = 3;
```

Comment:
Nit: I saw some value in having the "hex" qualifier that we're dropping with these changes, as the file could foreseeably include some other rounding to disambiguate, and also that a rounding precision of 3 is not just some arbitrary value we want to apply to all rounding, but is specifically focused around this need for lossless hex serialization.